submit_cgap/scripts/submit_ontology.py

```diff
@@ -41,8 +45,20 @@ def main(simulated_args_for_testing=None):
                 env=args.env,
                 validate_only=args.validate_only,
                 app=args.app,
-                submission_protocol=args.submission_protocol,
-        )
+                submission_protocol=args.submission_protocol)
+
+
+def verify_ontology_file(ontology_filename: str) -> bool:
+    if not os.path.exists(ontology_filename):
+        raise ScriptFailure(f"Cannot find specified ontology (JSON) file: {ontology_filename}")
+    try:
+        with io.open(ontology_filename, "r") as f:
+            ontology_json = json.load(f)
+            ontology_term_count = len(ontology_json["terms"])
+    except Exception:
+        raise ScriptFailure(f"Cannot load specified ontology (JSON) file: {ontology_filename}")
```

Comment:
I would try to report the error given back here, as it could be simple syntax formatting that may mean something to the user

Reply:
@willronchetti would it work for what you're asking to add
```
from dcicutils.misc_utils import get_error_message
```
at top of file and then do this here (untested):
```suggestion
    try:
        with io.open(ontology_filename, "r") as f:
            ontology_json = json.load(f)
            ontology_term_count = len(ontology_json["terms"])
    except Exception as e:
        raise ScriptFailure(f"Cannot load specified ontology (JSON) file {ontology_filename}."
                            f" {get_error_message(e)}")
```